doc/_static/mystyle.css

```diff
@@ -0,0 +1 @@
+@import url("https://fiware.org/style/fiware_readthedocs.css");
```

Comment:
Try using "https://www.fiware.org/style/fiware_readthedocs.css" (the former one redirects here). This is what we use in other components.